tmol/tests/score/ljlk/test_ljlk_energy_term.py

```diff
@@ -343,29 +344,65 @@ def test_whole_pose_scoring_module_smoke(rts_ubq_res, default_database, torch_de
     ljlk_energy.setup_poses(p1)
 
     ljlk_pose_scorer = ljlk_energy.render_whole_pose_scoring_module(p1)
-    # for ch in ljlk_pose_scorer.children():
-    #     print("child")
-    #     print(ch)
 
     coords = torch.nn.Parameter(p1.coords.clone())
-    scores = ljlk_pose_scorer(coords)
+    scores = ljlk_pose_scorer(
+        coords,
+    )
+
+    numpy.testing.assert_allclose(gold_vals, scores.cpu().detach().numpy(), atol=1e-4)
+
+
+def test_whole_pose_block_scoring(rts_ubq_res, default_database, torch_device):
+    gold_vals = numpy.array(
+        [
+            [
+                [
+                    [-3.9596581e-01, -5.5398142e-01, -5.9936708e-01, 0.0000000e00],
+                    [-5.5398142e-01, -3.7782270e-01, -8.2644725e-01, -7.1573496e-01],
+                    [-5.9936708e-01, -8.2644725e-01, -4.9665013e-01, -9.5627952e-01],
+                    [0.0000000e00, -7.1573496e-01, -9.5627952e-01, -6.3319176e-01],
+                ]
+            ],
+            [
+                [
+                    [2.0063031e-01, 3.8839228e-02, 9.2773259e-02, 0.0000000e00],
+                    [3.8839228e-02, 1.0832557e-01, 1.4488192e-01, 1.2620538e-04],
+                    [9.2773259e-02, 1.4488192e-01, 4.3663180e-01, 7.0729911e-02],
+                    [0.0000000e00, 1.2620538e-04, 7.0729911e-02, 7.5291798e-02],
+                ]
+            ],
+            [
+                [
+                    [1.8714617e-01, 4.1334108e-01, 5.5168569e-04, 0.0000000e00],
+                    [4.1334108e-01, 2.4108997e-01, 4.7006887e-01, 1.4917870e-01],
+                    [5.5168569e-04, 4.7006887e-01, 1.2823609e-01, 3.1399289e-01],
+                    [0.0000000e00, 1.4917870e-01, 3.1399289e-01, 3.6748153e-01],
+                ]
+            ],
+        ]
+    )
 
-    # make sure we're still good
```

Comment:
These arange calls are essentially a poor man's version of `cudaPeekAtLastError()` and are useful to add to unit tests after we run custom CUDA calls (which do not do any error checking... perhaps that should change)